cmd/rekor-server/app/serve.go

```diff
@@ -116,6 +121,13 @@ func init() {
 	serveCmd.Flags().String("signer-kmshash", "sha256", "hash algorithm used by the KMS")
 	serveCmd.Flags().String("signer-tink-kek-uri", "", "encryption key for decrypting Tink keyset. Valid options are [aws-kms://keyname, gcp-kms://keyname]")
 	serveCmd.Flags().String("signer-tink-keyset-path", "", "path to encrypted Tink keyset")
+
+	// tessera lifecycle configs
+	serveCmd.Flags().Uint("batch-max-size", tessera.DefaultBatchMaxSize, "the maximum number of entries that will accumulated before being sent to the sequencer")
+	serveCmd.Flags().Duration("batch-max-age", tessera.DefaultBatchMaxAge, "the maximum amount of time a batch of entries will wait before being sent to the sequencer")
+	serveCmd.Flags().Duration("checkpoint-interval", tessera.DefaultCheckpointInterval, "the frequency at which a checkpoint will be published")
```

Comment:
Is it somehow visible in CLI that these are duration strings that get parsed?

Reply:
```sh
start the Rekor server

Usage:
  rekor-server serve [flags]

Flags:
      --batch-max-age duration           the maximum amount of time a batch of entries will wait before being sent to the sequencer (default 250ms)
      --batch-max-size uint              the maximum number of entries that will accumulated before being sent to the sequencer (default 256)
      --checkpoint-interval duration     the frequency at which a checkpoint will be published (default 10s)
```

Reply:
with the example in the default value that seems clear enough